content/doc/book/using/_chapter.yml

```diff
@@ -8,3 +8,4 @@ sections:
   - change-time-zone
   - remote-access-api
   - executor-starvation
+  - using-nodes
```

Comment:
rename to "using-agents"? AFAICT the doc is solely about using agents (aka "remotely connected nodes")